src/event_key.rs

```diff
@@ -4,6 +4,7 @@
 
 use gdk_sys;
 use glib::translate::*;
+use libc;
```

Comment:
IMHO this use not used.

Reply:
https://travis-ci.org/gtk-rs/gdk/jobs/508079289#L673

Reply:
No it wasn't. Weird that I added it... Good catch!